R/hp_datatree.R

```diff
@@ -115,23 +115,25 @@ hp_datatree <- function(.data, title = "Hierplane", attributes = NULL, link = "t
 
   # Deal with duplicated child_id
   df$child <- gsub(".*[/]", "", df[["to"]])
-  # change all the `----` back to `/`
+  # Change all the placeholder tag back to `/`
   df$from <- gsub("~~~placeholder~~~", "/", df$from)
   df$to <- gsub("~~~placeholder~~~", "/", df$to)
   df$child <- gsub("~~~placeholder~~~", "/", df$child)
 
+  # Avoid displaying paths as links in case there are duplicates
+  link <- ifelse(link %in% "to", "child", link)
```

Comment:
I hope this change makes sense. I'm basically trying to avoid this from happening:
![image](https://user-images.githubusercontent.com/8585569/88623534-fd62d180-d059-11ea-81aa-d5a4fa89d395.png)

With this change it displays as originally intended (i think)
![image](https://user-images.githubusercontent.com/8585569/88623555-0bb0ed80-d05a-11ea-947e-438eeb6c4f97.png)

Reply:
Ya I noticed this as well. Thanks for fixing!